src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public String toString() {
+        return "[" + this.getStatusIcon() + "] " + this.description;
+    }
+}
```

Comment:
Good using of white spaces, makes the code very readable !

Reply:
I think overall, good use of the Egyptian style for the brackets. Maybe naming for the methods can be improved. Code is readable too!